evap/student/templates/student_index.html

```diff
@@ -16,7 +16,56 @@
             </div>
         </div>
     {% endif %}
-
+    {% if global_rewards %}
+        <div class="card mb-3">
+            <div class="card-header">
+                {% if global_rewards.next_reward_text %}
+                    {% blocktranslate with next_reward=global_rewards.next_reward_text %}Next reward: {{ next_reward }}{% endblocktranslate %}
+                {% else %}
+                    {% blocktranslate %}All rewards achieved{% endblocktranslate %}
+                {% endif %}
+            </div>
+            <div class="card-body">
+                <div>
+                    {% if global_rewards.next_reward_remaining_votes %}
+                        {% blocktranslate with remaining_votes=global_rewards.next_reward_remaining_votes %}{{ remaining_votes }} more evaluations required.{% endblocktranslate %}
+                    {% endif %}
+                    {% if global_rewards.last_vote_datetime %}
+                        {% blocktranslate trimmed with time_interval=global_rewards.last_vote_datetime|timesince %}
+                            The last evaluation was submitted {{time_interval}} ago.
+                        {% endblocktranslate %}
+                    {% endif %}
+                </div>
+                <div class="position-relative d-grid py-2 px-4">
+                    <span class="progress-container d-flex" style="height: 25px; grid-column-start: 1; grid-row-start: 1">
+                        <span class="progress m-0 bg-info">
+                            <span class="progress-bar progress-bar-striped progress-bar-animated bg-success
+                                width-percent-{% widthratio global_rewards.bar_width_votes global_rewards.max_reward_votes 100 %}"></span>
+                            <span class="progress-bartext">
+                                {{ global_rewards.current_votes }}/{{ global_rewards.max_reward_votes }}
+                            </span>
+                        </span>
+                    </span>
+                    {% for reward in global_rewards.rewards_with_progress %}
+                        <div class="position-relative "
```

Comment:
On a phone, the texts could very well overlap, not sure what an easy fix for this would be though - maybe the text could become smaller?

Reply:
UI/UX discussion with @janno42 resulted in: We are fine with that for v1. We likely want to have a second, more fancy version of the UI anyway. Resolving for now.